datascience-notebook/Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN npm install jupyterlab-flake8 \
     && jupyter labextension install jupyterlab-flake8 \
     && jupyter labextension install @ryantam626/jupyterlab_code_formatter \
     && jupyter serverextension enable --py jupyterlab_code_formatter
+    && pip install --upgrade nbstripout 
 
 # Fix conda install for NB_UID
 RUN chown -R $NB_UID /opt/conda \
- &&usermod -u $NB_UID $NB_USER
+ &&usermod -u $NB_UID $NB_USER
```

Comment:
looks like your editor is getting rid of the trailing newline at the end of files 😀